libs/coin-modules/coin-aptos/src/types/bridge.fixture.ts

```diff
@@ -0,0 +1,116 @@
+import BigNumber from "bignumber.js";
+import { faker } from "@faker-js/faker";
+import type { AptosAccount, AptosOperation, AptosOperationExtra, Transaction } from "./index";
+import { getAbandonSeedAddress } from "@ledgerhq/cryptoassets/abandonseed";
+import { getCryptoCurrencyById } from "@ledgerhq/cryptoassets/currencies";
+import { emptyHistoryCache } from "@ledgerhq/coin-framework/account/index";
+
+const currency = getCryptoCurrencyById("aptos");
```

Comment:
I think this file is also for Alpaca adaptation (but we can see if it tests things it is always good)